about.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CC Global Summit 2049</title>
+    <link rel="stylesheet" href="stylesAbout.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
+    <link href="media/favicon.ico" title="Icon" type="image/x-icon" rel="icon">
+</head>
+<body>
+    <header>
+        <section class="flexRow" id="top">
+            <img src="media/buttons/facebook.svg" alt="facebook handle">
+            <img src="media/buttons/twitter.svg" alt="twitter handle">
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+        </section>
+        <section class="flexRow" id="nav">
+            <a href="index.html" id="logo">
+                <img src="media/logo.png" alt="cc global logo">
+            </a>
+            <a href="about.html"><p>About</p></a>
+            <a href="#"><p>Program</p></a>
+            <a href="#"><p>Join</p></a>
+            <a href="#"><p>Sponsor</p></a>
+            <a href="#"><p>News</p></a>
+            <button>CC Campaign</button>
+        </section>
+    </header>
+    <section id="hero">
+        <img src="media/buttons/menu.svg" alt="menu" id="mobileMenuButton" onclick="mobileMenu()">
```

Comment:
To replicate your result, I had to view the project in a screen width of 333px, which is smaller than most mobile screen sizes other than the front screen of the galaxy fold, if I were to make the footer look the same as in the screenshot you called "ORIGINAL design", the text would be so small it would be impossible to read, so I believe this is an understandable design choice. If you view the project in a screen width of 497px or higher, you'll notice it looks the same as in the original design; in screens any smaller than that, the text will stack vertically for the user to be still able to read it.

Reply:
Hello @Davidosky007, you can be a bit lenient with the student concerning using exact font, mostly the Cocogoose font but Lato can easily be gotten from [Google Fonts ](https://fonts.google.com/specimen/Lato?query=lato). However, @AlexHamn the footer placement of text could still be achieved if you use @media screen size to reduce the font size on mobile view and increase the overall footer width or reduce the left and right padding/margin, and @AlexHamn your design should be consistent since you will not determine your target audience device, so as a developer you should be able to test it across all screens.

I hope my comments make sense.

Reply:
Yes, this makes a lot of sense; thank you for the explanation 🙏🏼 
I have improved the code to make the footer readjust when the screen goes beneath a 496px threshold; this is because that was the width at which problems started to appear.

![ksnip_20210812-115046](https://user-images.githubusercontent.com/81324033/129236988-7c42f47e-def8-4553-98cf-8b44d92ef96b.png)